docs/Developers/Newcomers_Guide.rst

```diff
@@ -383,6 +383,25 @@ and clicking ``New Pull Request``.
 
 **Congratulations!** You have now created your first ``Pull Request``!
 
+Now, before having other contributors review your work, the system will do an
+automated code review and will comment on your Pull Request.
+You may be required to enhance your Pull Request by modifying your branch on
+your fork.
+
+.. important::
+
+    It is **never** necessary to close your Pull Request. **Never!**
```

Comment:
Necessary doesn't feel like the proper word here. Maybe we can try `required` or `allowed` or something else you can think of

Reply:
I see your point, but in my opinion "necessary" sounds like the proper choice. I fell like the statement's purpose is to emphasize the fact that no matter what happens with a PR, the contributor doesn't have to close it. Sometimes, though, it is "required" to have a PR closed, when a maintainer asks you to. "Allowed" is a viable option, but it proposes a rather radical tonality. "Accepted" would be another option...What do you think?

Reply:
I think allowed is the best option among these. Accepted also doesn't go with the line.